### DISCUSSION TITLE
Understanding log messages

### DISCUSSION THREAD

**Discussion by charlieopp** [bluenviron/mediamtx#3581] [Questions]
### Question

I'm running mediamtx v1.8.3 on RPI 4/Bullseye. Recently upgraded from pre 1.0 that I'd been running successfully for a year. I've got a number of private clients (all hosts/clients on private net, no stun/ice servers) running chrome on Windows and others Webkit on IOS that are running 7x24. Because of a buffering backup problem on my camera (Reolink via RTSP/h264), I periodically need to reset the stream to keep it real-time. So every 15 minutes, I patch to another source for 1 second and switch back to the original source cam and that gets us back to real-time. Things usually work quite well (and flawlessly with early rev of mediamtx). But now at least once every 24 hours we get into a state where the displayed stream reports "Error stream not found. retrying in some seconds." This can be seen on some clients or all clients and the only way to get out of this is to restart everything. I've got a Python try/except on the patch which never traps an error. However, when the client is reporting stream not found, I see the following log messages which are a little hard to understand (note: removed many of the redundant stun/ice messages). All the server code runs on the same server (the 7.15 address).  Yet the log messages are reporting a patch problem that appears to be associated with client IP addresses (7.105 and 7.149). Incidentally, the flood of stun/ice messages also occurs when we get into this state. What are the sum of these messages telling us is going awry?

Thanks,
Charlie

Jul 28 23:07:20 raspbxhost mediamtx[337]: 2024/07/28 23:07:20 INF [WebRTC] [session acc415b1] created by 192.168.7.149:51295

Jul 28 23:07:20 raspbxhost mediamtx[337]: 2024/07/28 23:07:20 DEB [WebRTC] [session acc415b1] peer connection state: connecting

Jul 28 23:07:20 raspbxhost mediamtx[337]: 2024/07/28 23:07:20 DEB [WebRTC] [conn 192.168.7.149:51295] [s->c] HTTP/1.1 201 Created#012Accept-Patch: application/trickle-ice-sdpfrag#015#012Access-Control-Allow-Credentials: true#015#012Access-Control-Allow-Origin: *#015#012Access-Control-Expose-Headers: ETag, ID, Accept-Patch, Link, Location#015#012Content-Type: application/sdp#015#012Etag: *#015#012Id: acc415b1-bf9b-4b9d-8cea-98c114ba3e9c#015#012Link: <stun:stun.l.google.com:19302>; rel="ice-server"#015#012Link: <stun:stun.l.google.com:19302>; rel="ice-server"#015#012Link: <stun:stun.l.google.com:19302>; rel="ice-server"#015#012Link: <stun:stun.l.google.c

Jul 28 23:07:22 raspbxhost mediamtx[337]: 2024/07/28 23:07:22 DEB [WebRTC] [conn 192.168.7.149:51295] [c->s] OPTIONS /cam/whep HTTP/2.0#015#012Host: 192.168.7.15:8889#015#012Accept: */*#015#012Accept-Encoding: gzip, deflate, br, zstd#015#012Accept-Language: en-US,en;q=0.9#015#012Origin: https://192.168.7.15:8889#015#012Priority: u=1, i#015#012Referer: https://192.168.7.15:8889/cam/#015#012Sec-Ch-Ua: "Not/A)Brand";v="8", "Chromium";v="126", "Google Chrome";v="126"#015#012Sec-Ch-Ua-Mobile: ?0#015#012Sec-Ch-Ua-Platform: "Windows"#015#012Sec-Fetch-Dest: empty#015#012Sec-Fetch-Mode: cors#015#012Sec-Fetch-Site: same-origin#015#012User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36#015#012#015#012

Jul 28 23:07:22 raspbxhost mediamtx[337]: 2024/07/28 23:07:22 DEB [WebRTC] [conn 192.168.7.149:51295] [s->c] HTTP/1.1 204 No Content#012Access-Control-Allow-Credentials: true#015#012Access-Control-Allow-Headers: Authorization, Content-Type, If-Match#015#012Access-Control-Allow-Methods: OPTIONS, GET, POST, PATCH, DELETE#015#012Access-Control-Allow-Origin: *#015#012Access-Control-Expose-Headers: Link#015#012Link: <stun:stun.l.google.com:19302>; rel="ice-server"#015#012Link: <stun:stun.l.google.com:19302>; rel="ice-server"#015#012Link: <stun:stun.l.google.com:19302>; rel="ice-server"#015#012Link:  rel="ice-server"#015#012Link: <stun:stun.l.google.com:19302>; rel="ice-server"#015#012Lin

Jul 28 23:07:26 raspbxhost mediamtx[337]: 2024/07/28 23:07:26 DEB [WebRTC] [conn 192.168.7.105:53673] [c->s] PATCH /cam/whep/d9afa4b8-c2e7-4551-b530-84f5f6a56251 HTTP/2.0#015#012Host: 192.168.7.15:8889#015#012Accept: */*#015#012Accept-Encoding: gzip, deflate, br#015#012Accept-Language: en-US,en;q=0.9#015#012Cache-Control: no-cache#015#012Content-Length: 2109#015#012Content-Type: application/trickle-ice-sdpfrag#015#012If-Match: *#015#012Origin: https://192.168.7.15:8889#015#012Pragma: no-cache#015#012Referer: https://192.168.7.15:8889/cam/#015#012User-Agent: Mozilla/5.0 (iPad; CPU OS 15_8_2 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/15E148#015#012#015#012a=ice-ufrag:TgKK#015#012a=ice-pwd:I0tQrt9DBLdsQ9cG5jV7om0i#015#012m=video 9 UDP/TLS/RTP/SAVPF 96 97 98 99 100 101 102 125 104 124 106 107 108 109 127#015#012a=mid:0#015#012a=candidate:1108652525 1 udp 2122260223 88170b80-fc03-47f9-8420-0e1291ae4469.local 56166 typ host generation 0 ufrag TgKK network-id 1 network-cost 10#015#012a=candidate:1108652525 1 udp 2122260223 88170b80-fc03-47f9-8420-0e1291ae4469.local 62214 typ host generation 0 ufrag TgKK network-id 1 network-cost 10#015#012a=candidate:3277633369 1 udp 1686052607 209.65.188.172 62214 typ srflx raddr 0.0.0.0 rport 0 generation 0 ufrag TgKK network-id 1 network-cost 10#015#012a=candidate:3277633369 1 udp 1686052607 209.65.188.172 56166 typ srflx raddr 0.0.0.0 rport 0 generation 0 ufrag TgKK network-id 1 network-cost 10#015#012a=candidate:211242269 1 tcp 1518280447 88170b80-fc03-47f9-8420-0e1291ae4469.local 9 typ host tcptype active generation 0 ufrag TgKK network-id 1 network-cost 10#015#012a=candidate:211242269 1 tcp 1518280447 88170b80-fc03-47f9-8420-0e1291ae4469.local 9 typ host tcptype active generation 0 ufrag TgKK network-id 1 network-cost 10#015#012m=audio 9 UDP/TLS/RTP/SAVPF 111 63 103 9 0 8 105 13 110 113 126 118 119 112 113 114 115 116 117 120 121 122#015#012a=mid:1#015#012a=candidate:1108652525 1 udp 2122260223 88170b80-fc03-47f9-8420-0e1291ae4469.local 55039 typ host generation 0 ufrag TgKK network-id 1 network-cost 10#015#012a=candidate:1108652525 1 udp 2122260223 88170b80-fc03-47f9-8420-0e1291ae4469.local 54202 typ host generation 0 ufrag TgKK network-id 1 network-cost 10#015#012a=candidate:3277633369 1 udp 1686052607 209.65.188.172 3552 typ srflx raddr 0.0.0.0 rport 0 generation 0 ufrag TgKK network-id 1 network-cost 10#015#012a=candidate:3277633369 1 udp 1686052607 209.65.188.172 55039 typ srflx raddr 0.0.0.0 rport 0 generation 0 ufrag TgKK network-id 1 network-cost 10#015#012a=candidate:211242269 1 tcp 1518280447 88170b80-fc03-47f9-8420-0e1291ae4469.local 9 typ host tcptype active generation 0 ufrag TgKK network-id 1 network-cost 10#015#012a=candidate:211242269 1 tcp 1518280447 88170b80-fc03-47f9-8420-0e1291ae4469.local 9 typ host tcptype active generation 0 ufrag TgKK network-id 1 network-cost 10#015#012

Jul 28 23:07:26 raspbxhost mediamtx[337]: 2024/07/28 23:07:26 DEB [WebRTC] [conn 192.168.7.105:53673] [s->c] HTTP/1.1 404 Not Found#012Access-Control-Allow-Credentials: true#015#012Access-Control-Allow-Origin: *#015#012Content-Type: application/json; charset=utf-8#015#012Server: mediamtx#015#012#012(body of 29 bytes)

Jul 28 23:07:26 raspbxhost mediamtx[337]: 2024/07/28 23:07:26 DEB [WebRTC] [conn 192.168.7.105:53673] [c->s] DELETE /cam/whep/d9afa4b8-c2e7-4551-b530-84f5f6a56251 HTTP/2.0#015#012Host: 192.168.7.15:8889#015#012Accept: */*#015#012Accept-Encoding: gzip, deflate, br#015#012Accept-Language: en-US,en;q=0.9#015#012Content-Length: 0#015#012Origin: https://192.168.7.15:8889#015#012Referer: https://192.168.7.15:8889/cam/#015#012User-Agent: Mozilla/5.0 (iPad; CPU OS 15_8_2 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/15E148#015#012#015#012

Jul 28 23:07:26 raspbxhost mediamtx[337]: 2024/07/28 23:07:26 DEB [WebRTC] [conn 192.168.7.105:53673] [s->c] HTTP/1.1 404 Not Found#012Access-Control-Allow-Credentials: true#015#012Access-Control-Allow-Origin: *#015#012Content-Type: application/json; charset=utf-8#015#012Server: mediamtx#015#012#012(body of 29 bytes)

Jul 28 23:07:28 raspbxhost mediamtx[337]: 2024/07/28 23:07:28 DEB [WebRTC] [conn 192.168.7.105:53673] [c->s] OPTIONS /cam/whep HTTP/2.0#015#012Host: 192.168.7.15:8889#015#012Accept: */*#015#012Accept-Encoding: gzip, deflate, br#015#012Accept-Language: en-US,en;q=0.9#015#012Content-Length: 0#015#012Origin: https://192.168.7.15:8889#015#012Referer: https://192.168.7.15:8889/cam/#015#012User-Agent: Mozilla/5.0 (iPad; CPU OS 15_8_2 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/15E148#015#012#015#012

Jul 28 23:07:28 raspbxhost mediamtx[337]: 2024/07/28 23:07:28 DEB [WebRTC] [conn 192.168.7.105:53673] [s->c] HTTP/1.1 204 No Content#012Access-Control-Allow-Credentials: true#015#012Access-Control-Allow-Headers: Authorization, Content-Type, If-Match#015#012Access-Control-Allow-Methods: OPTIONS, GET, POST, PATCH, DELETE#015#012Access-Control-Allow-Origin: *#015#012Access-Control-Expose-Headers: Link#015#012Link: <stun:stun.l.google.com:19302>; rel="ice-server"#015#012Link:  rel="ice-server"#015#012Link: <stun:stun.l.google.com:19302>; rel="ice-server"#015#012Lin

Jul 28 23:07:28 raspbxhost mediamtx[337]: 2024/07/28 23:07:28 DEB [WebRTC] [conn 192.168.7.105:53673] [c->s] POST /cam/whep HTTP/2.0#015#012Host: 192.168.7.15:8889#015#012Accept: */*#015#012Accept-Encoding: gzip, deflate, br#015#012Accept-Language: en-US,en;q=0.9#015#012Content-Length: 5735#015#012Content-Type: application/sdp#015#012Origin: https://192.168.7.15:8889#015#012Referer: https://192.168.7.15:8889/cam/#015#012User-Agent: Mozilla/5.0 (iPad; CPU OS 15_8_2 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/15E148#015#012#015#012v=0#015#012o=- 2874941038418884878 2 IN IP4 127.0.0.1#015#012s=-#015#012t=0 0#015#012a=group:BUNDLE 0 1#015#012a=extmap-allow-mixed#015#012a=msid-semantic: WMS#015#012m=video 9 UDP/TLS/RTP/SAVPF 96 97 98 99 100 101 102 125 104 124 106 107 108 109 127#015#012c=IN IP4 0.0.0.0#015#012a=rtcp:9 IN IP4 0.0.0.0#015#012a=ice-ufrag:Lyg+#015#012a=ice-pwd:s9VjJmARCY/0zXlQA6/tuQpm#015#012a=ice-options:trickle#015#012a=fingerprint:sha-256 7F:E4:0C:49:57:F3:71:63:90:AB:87:16:73:11:C8:EE:E1:CF:02:78:37:00:28:6C:2A:DF:EE:A4:BF:0C:1F:D8#015#012a=setup:actpass#015#012a=mid:0#015#012a=extmap:1 urn:ietf:params:rtp-hdrext:toffset#015#012a=extmap:2 http://www.webrtc.org/experiments/rtp-hdrext/abs-send-time#015#012a=extmap:3 urn:3gpp:video-orientation#015#012a=extmap:4 http://www.ietf.org/id/draft-holmer-rmcat-transport-wide-cc-extensions-01#015#012a=extmap:5 http://www.webrtc.org/experiments/rtp-hdrext/playout-delay#015#012a=extmap:6 http://www.webrtc.org/experiments/rtp-hdrext/video-content-type#015#012a=extmap:7 http://www.webrtc.org/experiments/rtp-hdrext/video-timing#015#012a=extmap:8 http://www.webrtc.org/experiments/rtp-hdrext/color-space#015#012a=extmap:9 urn:ietf:params:rtp-hdrext:sdes:mid#015#012a=extmap:10 urn:ietf:params:rtp-hdrext:sdes:rtp-stream-id#015#012a=extmap:11 urn:ietf:params:rtp-hdrext:sdes:repaired-rtp-stream-id#015#012a=sendrecv#015#012a=msid:- fc373a1a-f6fc-4442-89f5-fadb1e34fbac#015#012a=rtcp-mux#015#012a=rtcp-rsize#015#012a=rtpmap:96 H264/90000#015#012a=rtcp-fb:96 goog-remb#015#012a=rtcp-fb:96 transport-cc#015#012a=rtcp-fb:96 ccm fir#015#012a=rtcp-fb:96 nack#015#012a=rtcp-fb:96 nack pli#015#012a=fmtp:96 level-asymmetry-allowed=1;packetization-mode=1;profile-level-id=640c1f#015#012a=rtpmap:97 rtx/90000#015#012a=fmtp:97 apt=96#015#012a=rtpmap:98 H264/90000#015#012a=rtcp-fb:98 goog-remb#015#012a=rtcp-fb:98 transport-cc#015#012a=rtcp-fb:98 ccm fir#015#012a=rtcp-fb:98 nack#015#012a=rtcp-fb:98 nack pli#015#012a=fmtp:98 level-asymmetry-allowed=1;packetization-mode=1;profile-level-id=42e01f#015#012a=rtpmap:99 rtx/90000#015#012a=fmtp:99 apt=98#015#012a=rtpmap:100 H264/90000#015#012a=rtcp-fb:100 goog-remb#015#012a=rtcp-fb:100 transport-cc#015#012a=rtcp-fb:100 ccm fir#015#012a=rtcp-fb:100 nack#015#012a=rtcp-fb:100 nack pli#015#012a=fmtp:100 level-asymmetry-allowed=1;packetization-mode=0;profile-level-id=640c1f#015#012a=rtpmap:101 rtx/90000#015#012a=fmtp:101 apt=100#015#012a=rtpmap:102 H264/90000#015#012a=rtcp-fb:102 goog-remb#015#012a=rtcp-fb:102 transport-cc#015#012a=rtcp-fb:102 ccm fir#015#012a=rtcp-fb:102 nack#015#012a=rtcp-fb:102 nack pli#015#012a=fmtp:102 level-asymmetry-allowed=1;packetization-mode=0;profile-level-id=42e01f#015#012a=rtpmap:125 rtx/90000#015#012a=fmtp:125 apt=102#015#012a=rtpmap:104 VP8/90000#015#012a=rtcp-fb:104 goog-remb#015#012a=rtcp-fb:104 transport-cc#015#012a=rtcp-fb:104 ccm fir#015#012a=rtcp-fb:104 nack#015#012a=rtcp-fb:104 nack pli#015#012a=rtpmap:124 rtx/90000#015#012a=fmtp:124 apt=104#015#012a=rtpmap:106 VP9/90000#015#012a=rtcp-fb:106 goog-remb#015#012a=rtcp-fb:106 transport-cc#015#012a=rtcp-fb:106 ccm fir#015#012a=rtcp-fb:106 nack#015#012a=rtcp-fb:106 nack pli#015#012a=fmtp:106 profile-id=0#015#012a=rtpmap:107 rtx/90000#015#012a=fmtp:107 apt=106#015#012a=rtpmap:108 red/90000#015#012a=rtpmap:109 rtx/90000#015#012a=fmtp:109 apt=108#015#012a=rtpmap:127 ulpfec/90000#015#012a=ssrc-group:FID 29284538 1036260000#015#012a=ssrc:29284538 cname:hq+tR4S6P+qs6J0n#015#012a=ssrc:29284538 msid:- fc373a1a-f6fc-4442-89f5-fadb1e34fbac#015#012a=ssrc:29284538 mslabel:-#015#012a=ssrc:29284538 label:fc373a1a-f6fc-4442-89f5-fadb1e34fbac#015#012a=ssrc:1036260000 cname:hq+tR4S6P+qs6J0n#015#012a=ssrc:1036260000 msid:- fc373a1a-f6fc-4442-89f5-fadb1e34fbac#015#012a=ssrc:1036260000 mslabel:-#015#012a=ssrc:1036260000 label:fc373a1a-f6fc-4442-89f5-fadb1e34fbac#015#012m=audio 9 UDP/TLS/RTP/SAVPF 111 63 103 9 0 8 105 13 110 113 126 118 119 112 113 114 115 116 117 120 121 122#015#012c=IN IP4 0.0.0.0#015#012a=rtcp:9 IN IP4 0.0.0.0#015#012a=ice-ufrag:Lyg+#015#012a=ice-pwd:s9VjJmARCY/0zXlQA6/tuQpm#015#012a=ice-options:trickle#015#012a=fingerprint:sha-256 7F:E4:0C:49:57:F3:71:63:90:AB:87:16:73:11:C8:EE:E1:CF:02:78:37:00:28:6C:2A:DF:EE:A4:BF:0C:1F:D8#015#012a=setup:actpass#015#012a=mid:1#015#012a=extmap:14 urn:ietf:params:rtp-hdrext:ssrc-audio-level#015#012a=extmap:2 http://www.webrtc.org/experiments/rtp-hdrext/abs-send-time#015#012a=extmap:4 http://www.ietf.org/id/draft-holmer-rmcat-transport-wide-cc-extensions-01#015#012a=extmap:9 urn:ietf:params:rtp-hdrext:sdes:mid#015#012a=extmap:10 urn:ietf:params:rtp-hdrext:sdes:rtp-stream-id#015#012a=extmap:11 urn:ietf:params:rtp-hdrext:sdes:repaired-rtp-stream-id#015#012a=sendrecv#015#012a=msid:- 59366b87-fda1-4c89-9c28-ed62bc48192a#015#012a=rtcp-mux#015#012a=rtpmap:111 opus/48000/2#015#012a=rtcp-fb:111 transport-cc#015#012a=fmtp:111 minptime=10;useinbandfec=1;stereo=1;sprop-stereo=1#015#012a=rtpmap:63 red/48000/2#015#012a=fmtp:63 111/111#015#012a=rtpmap:103 ISAC/16000#015#012a=rtpmap:9 G722/8000#015#012a=rtpmap:0 PCMU/8000#015#012a=rtpmap:8 PCMA/8000#015#012a=rtpmap:105 CN/16000#015#012a=rtpmap:13 CN/8000#015#012a=rtpmap:110 telephone-event/48000#015#012a=rtpmap:113 telephone-event/16000#015#012a=rtpmap:126 telephone-event/8000#015#012a=ssrc:983329696 cname:hq+tR4S6P+qs6J0n#015#012a=ssrc:983329696 msid:- 59366b87-fda1-4c89-9c28-ed62bc48192a#015#012a=ssrc:983329696 mslabel:-#015#012a=ssrc:983329696 label:59366b87-fda1-4c89-9c28-ed62bc48192a#015#012a=rtpmap:118 PCMU/8000/2#015#012a=rtcp-fb:118 transport-cc#015#012a=rtpmap:119 PCMA/8000/2#015#012a=rtcp-fb:119 transport-cc#015#012a=rtpmap:112 multiopus/48000/3#015#012a=fmtp:112 channel_mapping=0,2,1;num_streams=2;coupled_streams=1#015#012a=rtcp-fb:112 transport-cc#015#012a=rtpmap:113 multiopus/48000/4#015#012a=fmtp:113 channel_mapping=0,1,2,3;num_streams=2;coupled_streams=2#015#012a=rtcp-fb:113 transport-cc#015#012a=rtpmap:114 multiopus/48000/5#015#012a=fmtp:114 channel_mapping=0,4,1,2,3;num_streams=3;coupled_streams=2#015#012a=rtcp-fb:114 transport-cc#015#012a=rtpmap:115 multiopus/48000/6#015#012a=fmtp:115 channel_mapping=0,4,1,2,3,5;num_streams=4;coupled_streams=2#015#012a=rtcp-fb:115 transport-cc#015#012a=rtpmap:116 multiopus/48000/7#015#012a=fmtp:116 channel_mapping=0,4,1,2,3,5,6;num_streams=4;coupled_streams=4#015#012a=rtcp-fb:116 transport-cc#015#012a=rtpmap:117 multiopus/48000/8#015#012a=fmtp:117 channel_mapping=0,6,1,4,5,2,3,7;num_streams=5;coupled_streams=4#015#012a=rtcp-fb:117 transport-cc#015#012a=rtpmap:120 L16/8000/2#015#012a=rtcp-fb:120 transport-cc#015#012a=rtpmap:121 L16/16000/2#015#012a=rtcp-fb:121 transport-cc#015#012a=rtpmap:122 L16/48000/2#015#012a=rtcp-fb:122 transport-cc#015#012

Jul 28 23:07:28 raspbxhost mediamtx[337]: 2024/07/28 23:07:28 INF [WebRTC] [session 6c1ed41c] created by 192.168.7.105:53673

Jul 28 23:07:28 raspbxhost mediamtx[337]: 2024/07/28 23:07:28 DEB [WebRTC] [session 6c1ed41c] peer connection state: connecting

Jul 28 23:07:28 raspbxhost mediamtx[337]: 2024/07/28 23:07:28 DEB [WebRTC] [conn 192.168.7.105:53673] [s->c] HTTP/1.1 201 Created#012Accept-Patch: application/trickle-ice-sdpfrag#015#012Access-Control-Allow-Credentials: true#015#012Access-Control-Allow-Origin: *#015#012Access-Control-Expose-Headers: ETag, ID, Accept-Patch, Link, Location#015#012Content-Type: application/sdp#015#012Etag: *#015#012Id: 6c1ed41c-c564-4d23-b73d-c716c1ff821b#015#012Link: <stun:stun.l.google.com:19302>; rel="ice-server"#015#012Link: <stun:stun.l.google.com:19302>; rel="ice-server"#015#012Link: <stun:stun.l.google.c

Jul 28 23:07:30 raspbxhost mediamtx[337]: 2024/07/28 23:07:30 DEB [WebRTC] [session acc415b1] peer connection state: closed

Jul 28 23:07:30 raspbxhost mediamtx[337]: 2024/07/28 23:07:30 INF [WebRTC] [session acc415b1] closed: deadline exceeded while waiting connection

Jul 28 23:07:38 raspbxhost mediamtx[337]: 2024/07/28 23:07:38 DEB [WebRTC] [session 6c1ed41c] peer connection state: closed

Jul 28 23:07:38 raspbxhost mediamtx[337]: 2024/07/28 23:07:38 INF [WebRTC] [session 6c1ed41c] closed: deadline exceeded while waiting connection

**Comment by aler9** [comment]
Hello, from the logs it seems like some clients are having difficulty establishing a WebRTC peer connection with the server.

This might also depend on connectivity issues with the STUN server, that requires all clients (and the server) to be connected to the internet.

My advice is to disable the STUN server (by clearing `webrtcICEServers2`) and switch back to the original configuration. Eventually, fill `webrtcAdditionalHosts` with the IP of the server, as mentioned in the README:

https://github.com/bluenviron/mediamtx?tab=readme-ov-file#solving-webrtc-connectivity-issues

In this way you will remove the internet dependency and things will be more reliable.
